core/src/main/java/org/arquillian/smart/testing/Logger.java

```diff
@@ -0,0 +1,93 @@
+package org.arquillian.smart.testing;
+
+import java.util.logging.Level;
+
+import static java.lang.String.format;
+
+public class Logger {
+
+    private java.util.logging.Logger jul;
+
+    private static String prefix = "%s: Smart-Testing - ";
+
+    private Logger(java.util.logging.Logger logger) {
+        this.jul = logger;
+    }
+
+    public static Logger getLogger(Class clazz) {
```

Comment:
add javadoc to the factory methods. you could also use this shorter way of creating our Logger directly in the code below.